client/app/services/service-details/service-details.html

```diff
@@ -371,7 +371,9 @@ <h2 translate>Resources</h2>
         <div class="panel-body">
             <section>
                 <h2 translate>Relationships</h2>
-                <div class="container-fluid">
+                <pf-empty-state ng-if="!vm.service.parent_service || !vm.service.service_template" config="vm.emptyState"></pf-empty-state>
```

Comment:
`&&` not `||`

(because negating `a || b` gives `!a && !b` ;)

Reply:
(Unless the goal was really to display "No data available" even if only one of those is missing.)

Reply:
good call on the &&!!

updated the pr